### DISCUSSION TITLE
FIREWORKS!!!!

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#4877] [General]
Yeeee,

I have seen that in the new release we can finally customize, columns, I have not tested it yet so I might come back with comments, but I thought I would give all the dev team a big THANK YOUUUUUUU!!!!

![image](https://user-images.githubusercontent.com/82715690/157695168-e9fb540d-38b3-478d-8e11-f984c8bd5060.png)

Marco

**Comment by Topener** [comment]
We love to hear this! Very glad you like it, let me know if there's anything you're having issues with.
